src/shared/common/ScaleBar.js

```diff
@@ -6,12 +6,19 @@ import { Path, Shape, Surface, Text } from '@react-native-community/art';
 import GLOBAL from '../Globals';
 
 type Props = {
+    // whether to shift the scale bar a little bit up from the bottom
+    alignToBottom: boolean,
```

Comment:
when would this not be shifted a little bit? for "normal" projects?

Reply:
I don't shift the scale bar on building footprint with TMS imagery. it places the scale bar much too high in the image, which looks weird. I'll spare you the details of why it happens. I couldn't figure out an obvious way to unify the code, so that's what we have for now :)